pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -385,7 +385,6 @@ func (cb *ClusterBuilder) buildDefaultCluster(name string, discoveryType cluster
 		} else {
 			// Dual Stack
 			if features.EnableDualStack {
-				// If dual-stack, it may be [IPv4, IPv6] or [IPv6, IPv4]
 				// using Cluster_ALL to enable Happy Eyeballsfor upstream connections
 				c.DnsLookupFamily = cluster.Cluster_ALL
```

Comment:
Hi @howardjohn, I think the `DnsLookupFamily` is `cluster.Cluster_ALL` here, and the same as in `pkg/bootstrap/config.go`  L#153 and L#158  where I set to `option.DNSLookupFamily(option.DNSLookupFamilyIPS))`. And I remember that `DNSLookupFamilyIPS` should be the `Cluster_ALL`.  Please correct me if I am wrong.